charts/trento-server/Chart.yaml

```diff
@@ -7,7 +7,7 @@ description: The trento server chart contains all the components necessary to ru
 type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates
-version: 2.4.0-dev1
+version: 2.3.3-dev4
```

Comment:
Why this change from 2.4.0 to 2.3.3 ?

Reply:
Because it was accidentally changed to 2.4.0-dev1 instead of 2.3.3-dev3 in one previous PR